index.js

```diff
@@ -234,8 +237,8 @@ module.exports = postcss.plugin('postcss-css-variables', function(options) {
 		rulesThatHaveDeclarationsWithVariablesList.forEach(function(rule) {
 			var rulesToWorkOn = [].concat(rule);
 			// Split out the rule into each comma separated selector piece
-			// We only need to split if is actually comma separted(selectors > 1)
-			if(rule.selectors.length > 1) {
+			// We only need to split if it's actually a Rule with multiple selectors
+			if(rule.type == 'rule' && rule.selectors.length > 1) {
```

Comment:
Let's use `rule.type === 'rule'`

Also why are we moving away from `walkRules` if we then check for `rule.type === 'rule'` here anyway?

Reply:
It looks like it is because `walkRules` doesn't walk over `atrule` types.

Reply:
Sorry, I believed I had responded to this comment.  Yes, it's because `atrule` types were not being traversed by `walkRules`.